splash/qtutils.py

```diff
@@ -299,3 +321,47 @@ def qt_header_items(request_or_reply):
         (name, request_or_reply.rawHeader(name))
         for name in request_or_reply.rawHeaderList()
     ]
+
+
+def qt_send_key(key, target):
+    """
+    Send a key event that might be defined using emacs keyboard macro syntax
+    to target qt object. For example:
+        - RET -> Qt.Key_Return
+        - SPC -> Qt.Key_Space
+        - <FooBar> -> Qt.Key_FooBar
+    See: http://doc.qt.io/qt-5/qt.html#Key-enum for valid key types
+    """
+    # Try to match EDMACRO shortcut keys
+    key_type = EDMACRO_SHORTCUTS.get(key)
+    if key_type:
+        text = QT_KEY_INPUTS.get(key_type, '')
+        return qt_send_text(text, target, key_type)
+
+    # Try to match "<Key_Name>"
+    key_match = re.match(r'^\<(.+)\>$', key)
+    if key_match:
+        key_type = getattr(Qt, 'Key_%s' % key_match.group(1),
+                           Qt.Key_unknown)
```

Comment:
I think it is better to raise an error for unsupported keys, it should make debugging easier